Module documentation template (#8)
kyma-project/template-repository: docs/user/README.md

> > **NOTE:** This is a general template that can be used for a module README.md, example README.md, or any other README.md type in all module repositories in the Kyma organization. Use only those sections that suit your use case but keep the proposed section order.

Reviewer: I can use it as a root repo README? I don't think so. 

The template is very explicit a module overview template. How can I use it in other scenarios then? You mean I can take over the syntactical things like heading usage.. At the moment the readme is so small, I think the note does not bring value..

# {Module Name}
> Modify the title and insert the name of your module. Use Heading 1 (H1).

> ## {Overview}

Reviewer: Should be not a placeholder, should it?

> > **TIP:** Apart from the {Module Name} heading, you can use your own titles for the remaining sections.

Reviewer 2: I assume you can add other sections, too?

Reviewer 3: Looking at different modules `docs/user/README.md` files, the **Useful link**s and **Feedback** are not any kind of pattern so I guess we can treat them as examples. Sections and their titles differ so unless we want to unify them, let's make it clear that it's the module provider's choice.

> > **TIP:** Apart from the {Module Name} heading, you can use your own titles for the remaining sections. You can also add your own sections as you need to for your module.

Reviewer 2: ```suggestion
> **TIP:** Apart from the {Module Name} heading, you can use your own titles for the remaining sections. You can also add more module-specific sections.
```
